pkg/agreementreceiver/server.go

```diff
@@ -1,38 +1,72 @@
 // Copyright (C) 2018 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package agreementreceiver
+package bwagreement
```

Comment:
Rename the folder as well to keep it consistent with package name. Also is there a better name than `bwagreement`? I would call it `escrow`, because as I understand, this is the part that deals with settling. But maybe there is a better name for it.

Reply:
Changed the folder name to bwagreement. As there are few other satellite tickets referred to the bandwidth table as bwagreement.